index.js

```diff
@@ -46,8 +46,8 @@ const takeUntil = notifier => source => (start, sink) => {
       return;
     }
 
-    if (type === 2) notifierTalkback(2);
-    if (done === UNIQUE) sink(type, data);
```

Comment:
this seemed redundant as source should never emit after being disposed and whenever we change `done` we unsubscribe from the source

Reply:
I'm sorry it took so long :-) 
merged and published, thanks 👋🏻